contributing/code/standards.rst

```diff
@@ -172,6 +172,10 @@ Documentation
 
 * Add PHPDoc blocks for all classes, methods, and functions;
 
+* Group annotations together so that annotations of the same type immediately
```

Comment:
I just copied the text from [php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/7ef7ea94ccb0d528daa2f428226958e00926b928/README.rst#usage):
>phpdoc_separation [symfony]
Annotations in phpdocs should be grouped together so that annotations of the same type immediately follow each other, and annotations of a different type are separated by a single blank line.